src/rdkafka_assignor.h

```diff
@@ -66,15 +69,14 @@ typedef struct rd_kafka_assignor_s {
         rd_kafkap_str_t   *rkas_protocol_type;
         rd_kafkap_str_t   *rkas_protocol_name;
 
-        const void        *rkas_userdata;
-        size_t             rkas_userdata_size;
-
 	int                rkas_enabled;
 
+        int                rkas_supported_protocols;
```

Comment:
Add a doc-string saying what the expected flags are, e.g.:
`int           rkas_supported_porotocols;  /**< RD_KAFKA_ASSIGNOR_PROTOCOL_... */`